spec/spec_helper.rb

```diff
@@ -0,0 +1,5 @@
+require 'chefspec'
```

Comment:
They certainly would. I've updated the Gemfile.ci so Travis will have these Gems installed.

Thanks for catching that.